ugvc/mrd/mrd_utils.py

```diff
@@ -974,9 +981,10 @@ def generate_synthetic_signatures(
         trinuc_dict[trinucsub] = {}
         trinuc_dict[trinucsub]["index_to_sample"] = {}
         trinuc_dict[trinucsub]["trinuc_counter"] = {}
+        random_choice_replace = bool(n_subs_db < n_subs_signature)
```

Comment:
Nice. Where did you come across a signature larger than the db, btw?

You don't need the bool(), this does the same
random_choice_replace = n_subs_db < n_subs_signature

Reply:
In one of the Tempus patients, I think...don't fully remember.